code/routers/country_page/country.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express')
+const router = express.Router();
+
+const Country = require('../../models/country');
+
+const countryCodes = require('../../config/resCodes').country;
+
+
+/*
+  Insert country info api
+
+  inputs:
+    name: String
+    abbreviation: String
+    generalInformation: Collection
+    statistics: Collection
+
+  Inserts the country information into the country database schema
+  This is just a temporary api to insert data into the country schema
+*/
+
+
+router.post('/insert_country_info', (req, res) => {
+
+  const {name, abbreviation, generalInformation, statistics} = req.body
+
+  var userID;
+  // checks if user is logged in or external request
+  if ('userID' in req.body){
+    userID = req.body.userID;
+  } else if ('user' in req) {
+    userID = req.user._id;
+  } else {
+    return res.status(countryCodes.insertCountry.userNotGiven.status).send({
+      message: countryCodes.insertCountry.userNotGiven.message});
+  }
+
+
+  var newCountry = new Country();
+
+  newCountry.name = name;
+  newCountry.abbreviation = abbreviation;
+  newCountry.generalInformation = generalInformation;
+  newCountry.statistics = statistics;
+
+  newCountry.save(function(err, country){
+
+    if (err) {
+      return res.status(countryCodes.insertCountry.addCountryFail.status).send({
+        message: countryCodes.insertCountry.addCountryFail.message
+      });
+    } else {
+      return res.status(countryCodes.insertCountry.success.status).send({
+        message: countryCodes.insertCountry.success.message
+      });
+    }
+
+  });
+
+
+
+});
+/*
+  Country info api
+
+  inputs:
+    countryName: String
+
+  Gets the country info
+*/
+
+router.get('/get_country_info', (req, res) => {
```

Comment:
For the api call formatting can you change it to kabob case: get_country_info -> get-country-info

Reply:
Done.